es-models/ssm_centric/ssm_centric.mapping.yaml

```diff
@@ -1,5 +1,10 @@
 _all:
   enabled: false
+_size:
+  enabled: true
+_source:
+  excludes:
+  - __comment__
```

Comment:
It's ok to keep this like that, I mean `excludes: __comment__`.

When we build index for DAVE CA for production use we must prune `occurrence.*`, ie, `excludes: occurrence.*`.

Reply:
Yeah this PR is not for DAVE CA, this is for Jibboo